velocity/src/main/java/me/leoko/advancedban/velocity/command/CommandRecieverVelocity.java

```diff
@@ -0,0 +1,48 @@
+package me.leoko.advancedban.velocity.command;
+
+import com.google.inject.Inject;
+import com.velocitypowered.api.command.CommandSource;
+import com.velocitypowered.api.command.SimpleCommand;
+import com.velocitypowered.api.proxy.Player;
+import com.velocitypowered.api.proxy.ProxyServer;
+import me.leoko.advancedban.Universal;
+import me.leoko.advancedban.manager.CommandManager;
+import org.checkerframework.checker.nullness.qual.MonotonicNonNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class CommandRecieverVelocity implements SimpleCommand {
+
+  @Inject
+  private ProxyServer server;
+
+  @MonotonicNonNull private final String cmd;
+  @MonotonicNonNull private String[] args;
+  @MonotonicNonNull private CommandSource commandSource;
```

Comment:
I don't know where you got the idea that these should be field variables shared across invocations of `execute` and `suggest`. It looks like they should be local to both methods.